toxcore/Messenger.h

```diff
@@ -26,6 +26,7 @@
 #ifndef MESSENGER_H
 #define MESSENGER_H
 
+#include "tox.h"
```

Comment:
This creates an upward dependency. Generally, the solution is duplication and casting (I don't really like this solution) or using `int` and dealing with integer constants in the code (I don't like this solution either). Perhaps we can come up with a better solution?